default/techs.txt

```diff
@@ -2276,7 +2291,7 @@ Tech
     researchturns = 8
     prerequisites = [
         "SHP_MULTICELL_CAST"
-        "GRO_LIFECYCLE_MAN"
+        "SHP_MONSTER_MATURATION"
```

Comment:
Just noticed this, was going to setup a testgame but changing tech dependencies on something I know the AI uses isn't necessarily straightforward. I can (and will) test the changes but can't apply until @Dilvish-fo says it's OK.